sstables/time_window_compaction_strategy.hh

```diff
@@ -101,7 +101,8 @@ class time_window_compaction_strategy : public compaction_strategy_impl {
     time_window_compaction_strategy_options _options;
     int64_t _estimated_remaining_tasks = 0;
     db_clock::time_point _last_expired_check;
-    timestamp_type _highest_window_seen;
+    // As timestamp_type is an int64_t, a primitive type, it must be initialized here.
+    timestamp_type _highest_window_seen = 0;
     // Keep track of all recent active windows that still need to be compacted into a single SSTable
```

Comment:
Good catch. With _highest_window_seen left uninitialized, it may happen that the last window will not be actually considered last, so STCS is performed on the window rather than major compaction when it closes.

LGTM